### DISCUSSION TITLE
How to add custom icons.

### DISCUSSION THREAD

**Discussion by mildlyinteresting** [posquit0/Awesome-CV#408] [Q&A]
I tried many suggestions but failed to use my own icons. 
Is this even possible with overleaf?

**Comment by felixpradoh** [comment]
I have the same issue, I want to add a Kaggle icon and link but cannot :( 
I have tried `\kaggle{felixpradoh}` in the resume.tex file after adding

`% Defines writer's kaggle (optional)
% Usage: \kaggle{<kaggle name>}
\newcommand*{\kaggle}[1]{\def\@kaggle{#1}}`

in the awsome-cv.cls file

Does anyone know if it is possible to ad a Kaggle icon + link?
Thanks

**Comment by darshan315** [comment]
Maybe have a look [here](https://github.com/darshan315/Awesome-CV). Might be helpful.
